src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class Figure implements Drawable {
+    private int radius;
+    private String color;
+
+    public static class FigureSupplier {
+        private static final Random random = new Random();
+
+        protected int getRandomFigure() {
+            return random.nextInt(10) + 1;
+        }
+
+        public Circle getDefaultFigure() {
+            int radius = 10;
+            String color = "White";
```

Comment:
when you move your figure supplier, they should be DEFAULT_COLOR, DEFAULT_RADIUS constants